uint32 id
geometry_msgs/Vector3 position
geometry_msgs/Vector3 velocity
string type_name #ellipse_cylinder or cuboid
float64 param1 #minor semiradius or legnth
float64 param2 #major semiradius or width
float64 height #TBD